src/components/scripture/labels.ts

```diff
@@ -88,7 +91,16 @@ export const labelOfScriptureRanges = (refs: readonly ScriptureRange[]) => {
   const hasSame = (key: keyof ScriptureReference) =>
     uniq(refs.flatMap((ref) => [ref.start[key], ref.end[key]])).length === 1;
   const same = hasSame('book') ? 'book' : undefined;
-  const labels = refs.map((ref) => labelOfScriptureRange(ref, same)).join(', ');
+  const labels = refs.map((ref) => labelOfScriptureRange(ref, same));
```

Comment:
No need to create labels if we are going to drop them. I'd do the slice here.
```suggestion
  const labels = refs.slice(0, collapseAfter).map((ref) => labelOfScriptureRange(ref, same));
```